src/modules/tasklist.js

```diff
@@ -0,0 +1,21 @@
+class TaskList {
+  list = [];
```

Comment:
While you have done a fantastic job and the list is rendered correctly. 

The structure of the data used would not permit some of the requirements required for this milestone. Kindly consider having a list that is an array of object. 

![image](https://user-images.githubusercontent.com/27709832/228554672-941780d2-1c46-46c3-abb7-1eb2deefecd8.png)

This is because the list should be shorted by the index.

![image](https://user-images.githubusercontent.com/27709832/228553769-def016fa-19d0-4f1c-9a60-c5672a4e2081.png)

Reply:
Sorry,
I don't understand, because my implementation is exactly as you mentioned.

Reply:
The requirement suggests that the index is a value that belongs to the object of arrays and is numeric but you have used a loop to do this and that makes it difficult to order it as it is a loop.